x-pack/docs/en/ilm/pause-resume-ilm.asciidoc

```diff
@@ -1,4 +1,169 @@
 [[pause-resume-ilm]]
 == Pause and resume {ilm}
```

Comment:
I thnk we should change this tital to `Stop and Start ILM` since we changed the API names